project/SparkBuild.scala

```diff
@@ -968,6 +973,12 @@ object Volcano {
   )
 }
 
+object YuniKorn {
+  // Exclude all yunikorn file for Compile and Test
+  lazy val settings = Seq(
+    unmanagedSources / excludeFilter := HiddenFileFilter || "*YuniKorn*.scala")
```

Comment:
```suggestion
    unmanagedSources / excludeFilter += HiddenFileFilter || "*YuniKorn*.scala")
```

Looks like yunikorn setting overwrite accidently, try to use `+=` rather than `:=` in here.

[1] https://www.scala-sbt.org/1.x/docs/Appending-Values.html

Reply:
@Yikun it seems `volcano` needs the same improvement.